plugins/woocommerce/includes/gateways/paypal/class-wc-gateway-paypal.php

```diff
@@ -527,27 +527,31 @@ public function should_load() {
 		$should_load = $this->get_option( $option_key );
 
 		if ( '' === $should_load ) {
-
-			// New installs without PayPal Standard enabled don't load it.
-			if ( 'no' === $this->enabled && WC_Install::is_new_install() ) {
-				$should_load = false;
-			} else {
-				$should_load = true;
-			}
+			// Set default `_should_load` to 'yes' on existing stores with PayPal Standard enabled or with existing PayPal Standard orders.
+			$should_load = 'yes' === $this->enabled || $this->has_paypal_orders() ? true : false;
```

Comment:
This could be simplified with:
```
$should_load = 'yes' === $this->enabled || $this->has_paypal_orders();
```
But I understand if you want to keep the value more explicit to make it easier to follow.

Reply:
Thanks Wesley 😄 I realise this condition started off as something different and I didn't consider this much simpler form! Thanks for the code review.

I've fixed it in [1da3ddb](https://github.com/woocommerce/woocommerce/pull/47205/commits/1da3ddb3c068d5a00f895128751df6941d7dc903)